cirq/ops/controlled_gate.py

```diff
@@ -102,6 +96,15 @@ def __pow__(self, exponent: Any) -> 'ControlledGate':
             return NotImplemented
         return ControlledGate(new_sub_gate)
 
+    def _phase_by_(self, phase_turns: float, qubit_index: int):
+        if qubit_index == 0:
+            return self
+        phased_gate = protocols.phase_by(
+            self.sub_gate, phase_turns, qubit_index, None)
```

Comment:
Should this be "qubit_index-1"?

Reply:
Correct

Reply:
Make sure to include a test that exercises this case, e.g. a Controlled(Controlled(NotPhaseable)).